assignments/callbacks.js

```diff
@@ -23,25 +23,52 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 
 function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
+  // getLength passes the length of the array into the callback.\
+  cb(arr.length);
 }
 
+getLength(items, (lengthOfList) => {
+  console.log(lengthOfList);
+});
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  cb(arr[arr.length-1]);
 }
 
+last(items, (lastItem) => {
```

Comment:
this function is missing an = sign to work properly after the word 'last', try not to forget that when writing arrow functions or JavaScript will get mad 😢